ui/app/AppLayouts/Communities/controls/FeeRow.qml

```diff
@@ -0,0 +1,81 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+Control {
+    id: root
+
+    property string title
+    property string feeText
+
+    property bool highlightFee: false
+    property bool errorFee: false
+
+    background: null
+
+    contentItem: RowLayout {
+        StatusBaseText {
+            Layout.fillWidth: true
+            Layout.maximumWidth: (parent.width - parent.spacing) / 2
```

Comment:
```suggestion
           Binding on Layout.maximumWidth {
                value: (layout.width - layout.spacing) / 2
                delayed: true
            }
```

This is fixing the StackLayout warning I'm having.

Reply:
False alarm! It's not a proper fix. The layout for FeeRow and FeesPanel is broken

Reply:
Fixed by using Item on toplevel :)